docs/release-notes/index.rst

```diff
@@ -274,4 +278,7 @@ development keeps pace with the OS's new releases and support lifecycle.>
   We would like to express a big thank you to all the people who contributed to
   this release.
 
-  `@alex <>`_, `@blair <>`_, `@cam <>`_, `@devin <>`_
+  `@alex <>`_,
```

Comment:
I'm trying to remember why I used [literalrefs](https://github.com/canonical/rockcraft/blob/main/docs/release-notes/rockcraft-1-7-0.rst?plain=1#L135) here - Michael do you remember? I think it might've had to do with spellchecking

Reply:
Correct, it's so we don't have to add every contributor name to the allowlist. I also think it looks appropriate. It has similar appearance to mentions on GitHub and chat apps.

Reply:
what I mean is - shouldn't we also use literalrefs here on the template?

Reply:
Right, oops! [b496b53](https://github.com/canonical/starbase/pull/328/commits/b496b5303fa833dbdd5adad42d47a6ab18956005)